lib/rack/utils.rb

```diff
@@ -532,21 +532,35 @@ def context(env, app = @app)
       506 => 'Variant Also Negotiates',
       507 => 'Insufficient Storage',
       508 => 'Loop Detected',
-      509 => 'Bandwidth Limit Exceeded',
-      510 => 'Not Extended',
```

Comment:
I am okay to remove these. They are informally specified at best.

- `509`: https://http.dev/509
- `510`: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/510

However, can we check if they are being used?